internal/runners/initialize/rationalize.go

```diff
@@ -45,6 +48,13 @@ func rationalizeError(owner, project string, rerr *error) {
 			errs.SetInput(),
 		)
 
+	case errors.As(*rerr, &unrecognizedLanguageErr):
```

Comment:
Couldn't this function handle the error type from the language package?

Reply:
I thought the same, until I realized that the language package uses a function call to return a localized error, and not a specific error type.